### ISSUE TITLE
Can't .subscribeInvoices()

### ISSUE THREAD

**Issue by Doweig** [LN-Zap/node-lnd-grpc#63]
Hello guys,

I've recently started to use this module instead of the base grpc and I find it fantastic!
I'm having issues using the `subscribeInvoices` function however

Code like this does throws an error:
```
invoicesSub = Lightning.subscribeInvoices()
invoicesSub.on('data', onInvoiceData)
```

> TypeError: invoicesSub.on is not a function

that is because that function returns a Promise. However using `await` will block the execution forever, it will never reach the `.on()` statement and `.then` never gets called either.

Any idea how to solve that? Thank you in advance

PS: this code is after the `grpc.connect()` with a valid "active" status, and using lnd + proto 0.7

**Comment by mrfelton** [comment]
Hmm. Very interesting. I think you may have just found a bug! Working on a fix...

**Comment by mrfelton** [comment]
Fix included in [v0.2.16](https://github.com/LN-Zap/node-lnd-grpc/releases/tag/v0.2.16)

**Comment by Doweig** [comment]
That was fast! Thank you so much
